wfcGen.py

```diff
@@ -145,13 +143,13 @@ def getEntropy(self):
         if self.collapsed:
             return -1
         else:
-            if self.checkCollapsedNeighbours:
-                return len(cells.keys())
+            if self.collapsedNeighbours:
+                return 9223372036854775800 # max int - 1, this number need to be bigger than len of cells

```

Comment:
Firstly, that isn't technically guaranteed that a Python list length is less than int max (because Python has long arithmetics), although this situation appears extremely unlikely because I doubt anyone has this much RAM.

Still, how about
```suggestion
                return len(cells) + 1
```